generators/server/templates/src/test/java/package/web/rest/AccountResourceIT.java.ejs

```diff
@@ -23,6 +23,7 @@ package <%=packageName%>.web.rest;
 import <%=packageName%>.AbstractCassandraTest;
 <%_ } _%>
 import <%=packageName%>.<%= mainClass %>;
+import <%=packageName%>.config.TestSecurityConfiguration;
```

Comment:
Did you missed the condition if (authenticationType === 'oauth2')....?

Reply:
This isn't needed in this case because there's an `<%_ if (authenticationType === 'oauth2') { _%>` on line 20.

Reply:
But this condition ended in line 24, no?

Reply:
Forget, I read it bad :(